radio/src/lua/api_general.cpp

```diff
@@ -694,6 +694,30 @@ static int luaPlayTone(lua_State * L)
 }
 
 /*luadoc
```

Comment:
This block (and the function declaration) should be inside #if defined(HAPTIC), because some boards (i.e. X9D(not plus) has no HAPTIC